apps/docs/src/app/platform/component-docs/platform-object-list-item/platform-object-list-item-examples/platform-object-list-item-with-row-selection-example.component.ts

```diff
@@ -0,0 +1,135 @@
+import { Component } from '@angular/core';
+import { Observable, of } from 'rxjs';
+
+import { ListDataSource, DataProvider } from '@fundamental-ngx/platform';
+
+
+const LIST_ELEMENTS: Product[] = [
```

Comment:
please move to it to the own file and import it everywhere

Reply:
Each example has different values with different scenarios.